cpp/src/arrow/python/python-test.cc

```diff
@@ -383,5 +384,57 @@ TEST(PythonTest, ConstructStringArrayWithLeadingZeros) {
   ASSERT_OK(ConvertPySequence(list, pool, &out));
 }
 
+class NdarrayToArrowTest: public ::testing::Test {
```

Comment:
You'll probably get a format check error because of missing a space before the colon after `ArrowTest`.